client/blocks/login/error-notice.jsx

```diff
@@ -70,6 +72,27 @@ class ErrorNotice extends Component {
 
 		let message = error.message;
 
+		const { signupUrl } = this.props;
+
+		if ( error.code === 'unknown_user' && signupUrl ) {
+			message = this.props.translate(
+				"Hmm, we can't find a WordPress.com account for that social login. Please double check your information and try again." +
+					' Alternatively, you can {{a}}sign up for a new account{{/a}}.',
+				{
+					components: {
+						a: (
+							<a
+								href={ signupUrl }
+								onClick={ this.props.recordTracksEvent(
+									'calypso_login_social_unknown_user_signup_link_click'
```

Comment:
@vindl Are you fine with `calypso_login_social_unknown_user_signup_link_click` for this context?

Reply:
Yes. 👍

Reply:
Added to Tracks with 1005-gh-Automattic/tracks-events-registration